### ISSUE TITLE
Reorganization of issues

### ISSUE THREAD

**Issue by tesshucom** [airsonic/airsonic#1650]
It is difficult to use it effectively because the classification of issues is not maintained. Therefore, it will be organized.

 - Relabel to make it easier to find related things
 - The one with the label has priority. The one without the label is later.

____

The label of spring-framework is referenced. It is categorized by the prefix of for, in, status, and type. It doesn't need to be changed significantly and can be used with Airsonic with a few changes.

label | discription
-- | --
for: backport-to-x.x.x | Designates an issue for backport to x.x.x
for: external-project | Needs a fix in external project
for: merge-with-amendments | Needs some changes when we merge
for: stackoverflow | A question that's better suited to stackoverflow.com
for: team-attention | An issue we need to discuss as a team to make progress
in: core | Issues in core modules (aop, beans, core, context, expression)
in: data | Issues in data modules (jdbc, orm, oxm, tx)
in: messaging | Issues in messaging modules (jms, messaging)
in: test | Issues in the test module
in: web | Issues in web modules (web, webmvc, webflux, websocket)
status: backported | An issue that has been backported to maintenance branches
status: blocked | An issue that's blocked on an external project change
status: bulk-closed | An outdated, unresolved issue that's closed in bulk as part of a cleaning process
status: declined | A suggestion or change that we don't feel we should currently apply
status: duplicate | A duplicate of another issue
status: feedback-provided | Feedback has been provided
status: feedback-reminder | We've sent a reminder that we need additional information before we can continue
status: ideal-for-contribution | An issue that a contributor can help us with
status: invalid | An issue that we don't feel is valid
status: pending-design-work | Needs design work before any code can be developed
status: superseded | An issue that has been superseded by another
status: waiting-for-feedback | We need additional information before we can continue
status: waiting-for-triage | An issue we've not yet triaged or decided on
type: backport | An issue that is a backport of another issue to a maintenance branch
type: blocker | An issue that is blocking us from releasing
type: bug | A general bug
type: dependency-upgrade | A dependency upgrade
type: documentation | A documentation task
type: enhancement | A general enhancement
type: regression | A bug that is also a regression
type: task | A general task

https://github.com/spring-projects/spring-framework/labels

At least because it has a better perspective than the previous Airsonic.
I will start making changes as long as there is no objection. How about?
(If you wait for a response from someone who has not responded recently, it will never end.)

The precautions for replacement are classified and explained below.

**Comment by tesshucom** [comment]
#### Replace with new label

The following items may be exchanged as they are.
Naturally classified by prefix.

after | before
-- | --
for: backport-to-x.x.x | release
for: team-attention | discuss, organisation
status: bulk-closed | neverstale
status: duplicate | duplicate
status: ideal-for-contribution | good first issue, patches-welcome
status: pending-design-work | breaking change
status: waiting-for-feedback | research needed
status: waiting-for-triage | neverstale
type: documentation | documentation
type: enhancement | enhancement
type: regression | regression

**Comment by jooola** [comment]
neverstale label is used to prevent the stalebot from closing issues, this needs to be taken into count.

**Comment by tesshucom** [comment]
> neverstale label is used to prevent the stalebot from closing issues, this needs to be taken into count.

OK. Thank you!

**Comment by tesshucom** [comment]
####  reclassified to "in"

before
--
frontend
backend
browser specific
build
database
feature: docker
REST API
search
testsuite
translations
playback

The "in" of spring will be replaced with one that is compatible with Airsonic. This category is most often used on Airsonic labels. Therefore, it may be more helpful to add more items.

For example, transcoding, podcasts, web players, etc. should have separate categories.

**Comment by tesshucom** [comment]
#### Add Airsonic-specific items

before | Add to
-- | --
can't reproduce | Add new item to "staus"
feature request | Add new item to "type"
question | Add new item to "type"


Not in Spring.
In the case of Airsonic, it would be a little trouble without this.

**Comment by tesshucom** [comment]
#### Probably not necessary?

before
--
workaround
non trivial
crash
critical
low priority

It seems that it is appropriate to properly express in other items.
It's too abstract to understand the meaning.
It may include subjectivity and is not suitable for later use as a key for search classification.

Significance can be determined by customary immediate response and inclusion in the master, or by milestone.

**Comment by jooola** [comment]
> Use as is without changing
It should not be touched as it is used by the bot.

Actually no, it can be changed, but bots configurations needs to be changed as well, or you will end up with a bot mess like an avalanche of bot emails.

See https://github.com/airsonic/airsonic/blob/master/.github/stale.yml

I think the dependabot can use a local yaml configuration, it just needs to be created.

See https://dependabot.com/docs/config-file/

**Comment by tesshucom** [comment]
OK.
If you do not change it, there is no problem, right?
I will check the contents so that they can be used effectively.

**Comment by tesshucom** [comment]
OK.
I will check the contents so that they can be used effectively.

**Comment by tesshucom** [comment]
#### Isn't it better to use Projects in addition to Label?

##### security

Security does not exist in the label, but I think that it is suitable to add it to the "Projects" in addition to the usual classification of for, in, status, and type.

##### usability, code quality, performance, Hacktoberfest

These are a bit too abstract to use for classification. At least we need to relabel as for, in, status, and type.

The project already has the following items:

- Modernise and cleanup airsonic
- Improve the web interface

Most of them can be classified in this?
The term performance is generally treated as an incident, or a bug if performance requirements cannot be met. It seems that many other issues mention cleanup.

Focusing on the convenience of being easy to find later.
If you have any feedback!

**Comment by eharris** [comment]
I started this post off listing things that I noticed that didn't seem to apply or had other discrepancies, but I soon gave up because it was too time consuming to try to reconcile the back-and-forth discussion down to exactly what tags (and their replacements) are being proposed.

Since this is a discussion/proposal ticket, I suggest you provide (and maintain) a complete and definitive list of the "before" and "after" and "reasoning" for every Airsonic-specific label (including new and removed ones) being proposed.  Once that is done, it will be much easier to see what deficiencies might be remaining to be addressed.

**Comment by tesshucom** [comment]
OK.
I made the same attempt as you, and gave up. Think of the wreckage as "Replace with new label". That's the only items that can be transferred almost as they are.

 It may be a pain to hear, but Airsonic's current issus management is not that high quality. This means that it is difficult to move to general tracking with simple work. (That's why it's a meaningful job)

Most require reassignment (Rather, the classification has not even begun. We will now start a classification that was previously ignored).

That said, I don't like complete destruction.
The group at the bottom is the one that seems somewhat inadequate. They are proposing if they may be redistributed to the project.

Or, if those categories need to be expressed externally as Airsonic's mainly concerns, we can define new ones is "Projects". 
(However, some of them had been  probably improvised or set by people who are new to task classification, rather than called categories.)

____

For the full migration list, either is fine, if it's a consensus that you need it, you should do so. I think that... If the original issue management is high quality and reliable, it is definitely worth it. Otherwise it's a waste of time and just a blocker. We ask other members for their views.

**Comment by fxthomas** [comment]
This sounds good, :+1: for doing that!

A few comments:

* `in: messaging` might be a bit useless for Airsonic?
* I would try to transform some of the current labels (`playback`, `docker`, `rest`) into `in: xxx`.

**Comment by tesshucom** [comment]
We may leave the in: message.
Exception message and translation has only not been treated carefully until now.
There seems to be an update that was originally supposed to be managed.(This is intuitive)
Also, I do not know the details yet, but it may be possible to work with #1589.
The decision may be a little later.

in: xxx needs to be reconfigured. There may be a JDK, a startup, etc.
It should be possible to improve this by repeating the assignment process several times.
While it is a technically relevant classification, it is also important not to increase the number of categories too much.
(Although it contradicts)

For example, the image below : 

![image](https://user-images.githubusercontent.com/27724847/80281316-39c61f00-8745-11ea-9814-4661a31fe0d6.png)

Compound search is easy if it is classified correctly, right?
(But if you divide it too much, the list will be difficult to use!)

It's hard to decide everything at once, but we can brush up gradually.

**Comment by fxthomas** [comment]
Ah, `in: message` was about translations? That wasn't how i read it, I thought it was related to another component. Might be better to replace it by `in: translations` then?

Other than that let's go for it, feel free to do it any time!

**Comment by tesshucom** [comment]
OK.
I'm not stuck with SpringStyle's ``in:``, and I think it should be optimized.
There may be many ``in:`` created at first, but think of it as a transitional period.
I think it is good to classify and use them, and to think about the right classification while considering the amount.
Probably somewhere there is a level that users and bug trackers are generally satisfied with.

**Comment by fxthomas** [comment]
Nice, thank you! It looks clean at first glance, I'll put suggestions if I have them here.

**Comment by tesshucom** [comment]
Maintenance query sample.

## A label that is always given

#### Issue type
issues are classified into one of the following types. 

<table>
	<tbody>
		<tr>
			<th>prefix</th>
			<th>name</th>
			<th>"In" required</th>
		</tr>
		<tr>
			<td>for</td>
			<td>team-attention</td>
			<td></td>
		</tr>
		<tr>
			<td rowspan="8">type</td>
			<td>regression</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>bug</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>documentation</td>
			<td></td>
		</tr>
		<tr>
			<td>enhancement</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>question</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>request</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>tips</td>
			<td>〇</td>
		</tr>
		<tr>
			<td>task</td>
			<td></td>
		</tr>
	</tbody>
</table>

[Uncategorized issues](https://github.com/airsonic/airsonic/issues?q=is%3Aopen+is%3Aissue+-label%3A%22type%3A+question%22+-label%3A%22type%3A+bug%22+-label%3A%22type%3A+documentation%22+-label%3A%22for%3A+team-attention%22+-label%3A%22type%3A+enhancement%22+-label%3A%22type%3A+tips%22+-label%3A%22type%3A+regression%22+-label%3A%22type%3A+task%22+-label%3A%22type%3A+request%22+)

#### in:

Below issues with the prefix ``type`` are always labeled ``in:``.

label | in:
-- | --
type: regression | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=is%3Aopen+label%3A%22type%3A+regression%22+-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)
type: bug | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=is+open+label%3A%22type%3A+bug%22++-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)
type: enhancement | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=label%3A%22type%3A+enhancement%22+-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)
type: question | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=is%3Aopen+label%3A%22type%3A+question%22+-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)
type: request | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=is%3Aopen+label%3A%22type%3A+request%22+-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)
type: tips | [Uncategorized](https://github.com/airsonic/airsonic/issues?q=label%3A%22type%3A+tips%22+-label%3A%22in%3A+build%22+-label%3A%22in%3A+cover+art%22+-label%3A%22in%3A+data%22+-label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22in%3A+docker%22+-label%3A%22in%3A+external+cooperation%22+-label%3A%22in%3A+jdk%22+-label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22in%3A+rest%22+-label%3A%22in%3A+search%2Fsort%22+-label%3A%22in%3A+smart+playlists%22+-label%3A%22in%3A+tagger%22+-label%3A%22in%3A+test%22+-label%3A%22in%3A+transcoding%22+-label%3A%22in%3A+translations%22+-label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22in%3A+web%22+-label%3A%22in%3A+web-playback%22+)

The rest is status ...
_______


By applying this, you can create a matrix of issues that users and developers can use.


  | type: bug | type : question | type: request
-- | -- | -- | --
in: build |   |   |  
in: data |   |   |  
in: dlna/sonos |   |   |  
in: docker |   |   |  
in: external cooperation |   |   |  
in: jdk |   |   |  
in: podcast |   |   |  
in: rest |   |   |  
in: search |   |   |  
in: test |   |   |  
in: transcoding |   |   |  
in: translations |   |   |  
in: UNKNOWN-backend |   |   |  
in: UNKNOWN-frontend |   |   |  
in: web-playback |   |   |  
in: web |   |   |  

You can create a status link in the cell.

**Comment by jooola** [comment]
As little side note, I recommend using the embed GitHub auto label tool that comes with issue templates.

You can auto label new issue based on the kind of issue someone created (bug/feature request/...).

Here is the doc: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Here is an example: https://github.com/directus/gatsby-source/tree/master/.github/ISSUE_TEMPLATE

**Comment by tesshucom** [comment]
That's good! I was hoping that I could do it.
I will check the content and reflect it.

In fact, I find it difficult to operate like a full-scale management product. (Functional and personnel) Still, you may be able to do something similar to that in part.

**Comment by tesshucom** [comment]
Summary example. There is also the idea of ​​creating and pin such a link.


### Main opened issues recently updated

<table><tbody><tr><th>regression</th><th>bug</th><th>question</th><th>request</th></tr><tr><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr></tbody></table>





<details>
<summary>Details by "in"</summary>
<table><tbody><tr><td colspan="2" rowspan="2"></td><th colspan="4">type</th></tr><tr><th>regression</th><th>bug</th><th>question</th><th>request</th></tr><tr><th rowspan="19">in</th><th>build</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+build%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+build%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+build%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+build%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+build%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+build%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+build%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+build%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>cover art</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+cover+art%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+cover+art%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+cover+art%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+cover+art%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+cover+art%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+cover+art%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+cover+art%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+cover+art%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>data</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+data%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+data%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+data%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+data%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+data%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+data%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+data%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+data%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>dlna/sonos</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+dlna%2Fsonos%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+dlna%2Fsonos%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+dlna%2Fsonos%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+dlna%2Fsonos%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+dlna%2Fsonos%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>docker</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+docker%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+docker%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+docker%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+docker%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+docker%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+docker%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+docker%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+docker%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>external cooperation</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+external+cooperation%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+external+cooperation%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+external+cooperation%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+external+cooperation%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+external+cooperation%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+external+cooperation%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+external+cooperation%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+external+cooperation%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>jdk</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+jdk%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+jdk%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+jdk%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+jdk%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+jdk%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+jdk%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+jdk%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+jdk%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>podcast/radio</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+podcast%2Fradio%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+podcast%2Fradio%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+podcast%2Fradio%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+podcast%2Fradio%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+podcast%2Fradio%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>rest</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+rest%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+rest%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+rest%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+rest%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+rest%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+rest%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+rest%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+rest%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>search/sort</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+search%2Fsort%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+search%2Fsort%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+search%2Fsort%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+search%2Fsort%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+search%2Fsort%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+search%2Fsort%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+search%2Fsort%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+search%2Fsort%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>smart playlists</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+smart+playlists%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+smart+playlists%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+smart+playlists%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+smart+playlists%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+smart+playlists%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+smart+playlists%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+smart+playlists%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+smart+playlists%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>tagger</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+tagger%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+tagger%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+tagger%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+tagger%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+tagger%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+tagger%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+tagger%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+tagger%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>test</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+test%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+test%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+test%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+test%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+test%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+test%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+test%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+test%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>transcoding</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+transcoding%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+transcoding%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+transcoding%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+transcoding%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+transcoding%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+transcoding%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+transcoding%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+transcoding%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>translations</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+translations%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+translations%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+translations%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+translations%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+translations%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+translations%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+translations%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+translations%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>UNKNOWN-backend</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+UNKNOWN-backend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+UNKNOWN-backend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+UNKNOWN-backend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+UNKNOWN-backend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+UNKNOWN-backend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>UNKNOWN-frontend</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+UNKNOWN-frontend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+UNKNOWN-frontend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+UNKNOWN-frontend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+UNKNOWN-frontend%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+UNKNOWN-frontend%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>web-playback</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+web-playback%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+web-playback%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+web-playback%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+web-playback%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+web-playback%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+web-playback%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+web-playback%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+web-playback%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr><tr><th>web</th><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+web%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+regression%22label%3A%22in%3A+web%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+web%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22label%3A%22in%3A+web%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+web%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+question%22label%3A%22in%3A+web%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td><td><a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+web%22+-label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="without valid feedback">:red_circle:</a> <a href="https://github.com/airsonic/airsonic/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+request%22label%3A%22in%3A+web%22+label%3A%22status%3A+feedback-provided%22+sort%3Aupdated-desc" title="with valid feedback">:white_circle:</a> </td></tr></tbody></table>


</details>

**Comment by fxthomas** [comment]
> Summary example. There is also the idea of ​​creating and pin such a link.

I assume this will not be automatically updated? In any case I like the idea, might help with searching issues!

**Comment by tesshucom** [comment]
The link is a Github URL query.
So if the issue status changes, it will be reflected in the appropriate classification.

Note that the displayed results are sorted in recent ascending order.
"Recently posted by someone" and "Recently modified by someone" are displayed above.

It can be used effectively with the correction of #1707.

The table is not automatically possible ...
I wrote a nice program.:disappointed:
If the "in" classification doesn't change much, you can use it fixed.

**Comment by tesshucom** [comment]
@airsonic/maintainers 

The following labels have moved to https://github.com/airsonic/airsonic/projects.
You can easily browse by the reference function of Github.

 - Hacktobefest
 - usability
 - quality
 - performance
 - workaround
 - trivial
 - low priority

This does not mean moving to the so-called trash. I think it is meaningful as a classification at the time it was created. Therefore I kept it so that I could refer to the original.
However, a few years ago... the situation has changed dramatically since the days when it was easy to unify the goals right after the subsonic branch. The current situation is a mix of old bugs, recent bugs and many requests. As you can see.
If we manage issues in the long run, we can make small changes to a better management method regularly. Keep in mind that the current method may not be the best, but such a transformation may be necessary.

@muff1nman

Where is airsonic heading? It's been controlled by fxthomas for a while, so it's quite a step forward. His work was more than just coding and reviewing. Some complained about speed, but I don't think so at all.
However, it is currently stagnant. It has some fatal bugs and the master build is dead. I can do a little if I need help, but I don't think changing the master arbitrarily at one's discretion is desired. The triage and merge criteria were previously ambiguous, and everything would stop working unless there were at least multiple active members.
I support a development style that does not force anything. But now it seems little too unclear.
